kubernetes-gcp-typescript/package.json

```diff
@@ -0,0 +1,11 @@
+{
+		"name": "tmp",
+		"devDependencies": {
+			"@types/node": "^14"
+		},
+		"dependencies": {
+			"typescript": "^4.0.0",
+			"@pulumi/pulumi": "^3.0.0",
+			"@pulumi/gcp": "6.39.0"
+		}
+}
```

Comment:
```suggestion
{
    "name": "tmp",
    "devDependencies": {
        "@types/node": "^14"
    },
    "dependencies": {
        "typescript": "^4.0.0",
        "@pulumi/pulumi": "^3.0.0",
        "@pulumi/gcp": "6.39.0"
    }
}
```

Reply:
(There's an [open issue](https://github.com/pulumi/pulumi/issues/10776) on the codegen for this.)

Reply:
Fixed in 440fb15